Gems/Terrain/Code/Source/Components/TerrainPhysicsColliderComponent.cpp

```diff
@@ -82,6 +83,8 @@ namespace Terrain
                     ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
                     ->Attribute(AZ::Edit::Attributes::Visibility, AZ::Edit::PropertyVisibility::ShowChildrenOnly)
                     ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
+                    ->DataElement(AZ::Edit::UIHandlers::Default, &TerrainPhysicsColliderConfig::m_defaultMaterialSelection,
+                        "Default Surface Physics Material", "Select a material to be used by maps surfaces by default")
```

Comment:
"to be used by maps surfaces by default" -> 
maybe either "to be used as the default surface material" or "to be used by unmapped surfaces by default"?

Reply:
I like @mbalfour-amzn suggestion "to be used by unmapped surfaces by default", it explicitly says when the default material will be used.

Reply:
Nice, updated the wording, thanks!